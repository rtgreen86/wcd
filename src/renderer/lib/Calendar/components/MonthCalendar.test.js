import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Month from './MonthCalendar';

describe('DaysGrid', () => {
  describe.each([
    [2000, 1],
    [2000, 2],
    [2000, 4],
    [2001, 1],
    [2001, 2],
    [2001, 4],
  ])('rendered with %i-%i', (year, month) => {
    test('match snapshot', function () {
      const { asFragment } = render(<Month year={year} month={month} />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
