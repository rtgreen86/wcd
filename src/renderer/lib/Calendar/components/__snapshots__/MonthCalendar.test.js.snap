// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DaysGrid rendered with 2000-1 match snapshot 1`] = `
<DocumentFragment>
  <table>
    <caption>
      January
    </caption>
    <thead>
      <tr>
        <td
          class="weekend"
        >
          Sun
        </td>
        <td
          class=""
        >
          Mon
        </td>
        <td
          class=""
        >
          Tue
        </td>
        <td
          class=""
        >
          Wed
        </td>
        <td
          class=""
        >
          Thu
        </td>
        <td
          class=""
        >
          Fri
        </td>
        <td
          class="weekend"
        >
          Sat
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td />
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class=""
        >
          3
        </td>
        <td
          class=""
        >
          4
        </td>
        <td
          class="weekend"
        >
          5
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          6
        </td>
        <td
          class=""
        >
          7
        </td>
        <td
          class=""
        >
          8
        </td>
        <td
          class=""
        >
          9
        </td>
        <td
          class=""
        >
          10
        </td>
        <td
          class=""
        >
          11
        </td>
        <td
          class="weekend"
        >
          12
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          13
        </td>
        <td
          class=""
        >
          14
        </td>
        <td
          class=""
        >
          15
        </td>
        <td
          class=""
        >
          16
        </td>
        <td
          class=""
        >
          17
        </td>
        <td
          class=""
        >
          18
        </td>
        <td
          class="weekend"
        >
          19
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          20
        </td>
        <td
          class=""
        >
          21
        </td>
        <td
          class=""
        >
          22
        </td>
        <td
          class=""
        >
          23
        </td>
        <td
          class=""
        >
          24
        </td>
        <td
          class=""
        >
          25
        </td>
        <td
          class="weekend"
        >
          26
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          27
        </td>
        <td
          class=""
        >
          28
        </td>
        <td
          class=""
        >
          29
        </td>
        <td />
        <td />
        <td />
        <td />
      </tr>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`DaysGrid rendered with 2000-2 match snapshot 1`] = `
<DocumentFragment>
  <table>
    <caption>
      February
    </caption>
    <thead>
      <tr>
        <td
          class="weekend"
        >
          Sun
        </td>
        <td
          class=""
        >
          Mon
        </td>
        <td
          class=""
        >
          Tue
        </td>
        <td
          class=""
        >
          Wed
        </td>
        <td
          class=""
        >
          Thu
        </td>
        <td
          class=""
        >
          Fri
        </td>
        <td
          class="weekend"
        >
          Sat
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td />
        <td />
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class=""
        >
          3
        </td>
        <td
          class="weekend"
        >
          4
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          5
        </td>
        <td
          class=""
        >
          6
        </td>
        <td
          class=""
        >
          7
        </td>
        <td
          class=""
        >
          8
        </td>
        <td
          class=""
        >
          9
        </td>
        <td
          class=""
        >
          10
        </td>
        <td
          class="weekend"
        >
          11
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          12
        </td>
        <td
          class=""
        >
          13
        </td>
        <td
          class=""
        >
          14
        </td>
        <td
          class=""
        >
          15
        </td>
        <td
          class=""
        >
          16
        </td>
        <td
          class=""
        >
          17
        </td>
        <td
          class="weekend"
        >
          18
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          19
        </td>
        <td
          class=""
        >
          20
        </td>
        <td
          class=""
        >
          21
        </td>
        <td
          class=""
        >
          22
        </td>
        <td
          class=""
        >
          23
        </td>
        <td
          class=""
        >
          24
        </td>
        <td
          class="weekend"
        >
          25
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          26
        </td>
        <td
          class=""
        >
          27
        </td>
        <td
          class=""
        >
          28
        </td>
        <td
          class=""
        >
          29
        </td>
        <td
          class=""
        >
          30
        </td>
        <td
          class=""
        >
          31
        </td>
        <td />
      </tr>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`DaysGrid rendered with 2000-4 match snapshot 1`] = `
<DocumentFragment>
  <table>
    <caption>
      April
    </caption>
    <thead>
      <tr>
        <td
          class="weekend"
        >
          Sun
        </td>
        <td
          class=""
        >
          Mon
        </td>
        <td
          class=""
        >
          Tue
        </td>
        <td
          class=""
        >
          Wed
        </td>
        <td
          class=""
        >
          Thu
        </td>
        <td
          class=""
        >
          Fri
        </td>
        <td
          class="weekend"
        >
          Sat
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class=""
        >
          3
        </td>
        <td
          class=""
        >
          4
        </td>
        <td
          class=""
        >
          5
        </td>
        <td
          class="weekend"
        >
          6
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          7
        </td>
        <td
          class=""
        >
          8
        </td>
        <td
          class=""
        >
          9
        </td>
        <td
          class=""
        >
          10
        </td>
        <td
          class=""
        >
          11
        </td>
        <td
          class=""
        >
          12
        </td>
        <td
          class="weekend"
        >
          13
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          14
        </td>
        <td
          class=""
        >
          15
        </td>
        <td
          class=""
        >
          16
        </td>
        <td
          class=""
        >
          17
        </td>
        <td
          class=""
        >
          18
        </td>
        <td
          class=""
        >
          19
        </td>
        <td
          class="weekend"
        >
          20
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          21
        </td>
        <td
          class=""
        >
          22
        </td>
        <td
          class=""
        >
          23
        </td>
        <td
          class=""
        >
          24
        </td>
        <td
          class=""
        >
          25
        </td>
        <td
          class=""
        >
          26
        </td>
        <td
          class="weekend"
        >
          27
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          28
        </td>
        <td
          class=""
        >
          29
        </td>
        <td
          class=""
        >
          30
        </td>
        <td
          class=""
        >
          31
        </td>
        <td />
        <td />
        <td />
      </tr>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`DaysGrid rendered with 2001-1 match snapshot 1`] = `
<DocumentFragment>
  <table>
    <caption>
      January
    </caption>
    <thead>
      <tr>
        <td
          class="weekend"
        >
          Sun
        </td>
        <td
          class=""
        >
          Mon
        </td>
        <td
          class=""
        >
          Tue
        </td>
        <td
          class=""
        >
          Wed
        </td>
        <td
          class=""
        >
          Thu
        </td>
        <td
          class=""
        >
          Fri
        </td>
        <td
          class="weekend"
        >
          Sat
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class="weekend"
        >
          3
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          4
        </td>
        <td
          class=""
        >
          5
        </td>
        <td
          class=""
        >
          6
        </td>
        <td
          class=""
        >
          7
        </td>
        <td
          class=""
        >
          8
        </td>
        <td
          class=""
        >
          9
        </td>
        <td
          class="weekend"
        >
          10
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          11
        </td>
        <td
          class=""
        >
          12
        </td>
        <td
          class=""
        >
          13
        </td>
        <td
          class=""
        >
          14
        </td>
        <td
          class=""
        >
          15
        </td>
        <td
          class=""
        >
          16
        </td>
        <td
          class="weekend"
        >
          17
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          18
        </td>
        <td
          class=""
        >
          19
        </td>
        <td
          class=""
        >
          20
        </td>
        <td
          class=""
        >
          21
        </td>
        <td
          class=""
        >
          22
        </td>
        <td
          class=""
        >
          23
        </td>
        <td
          class="weekend"
        >
          24
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          25
        </td>
        <td
          class=""
        >
          26
        </td>
        <td
          class=""
        >
          27
        </td>
        <td
          class=""
        >
          28
        </td>
        <td />
        <td />
        <td />
      </tr>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`DaysGrid rendered with 2001-2 match snapshot 1`] = `
<DocumentFragment>
  <table>
    <caption>
      February
    </caption>
    <thead>
      <tr>
        <td
          class="weekend"
        >
          Sun
        </td>
        <td
          class=""
        >
          Mon
        </td>
        <td
          class=""
        >
          Tue
        </td>
        <td
          class=""
        >
          Wed
        </td>
        <td
          class=""
        >
          Thu
        </td>
        <td
          class=""
        >
          Fri
        </td>
        <td
          class="weekend"
        >
          Sat
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class="weekend"
        >
          3
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          4
        </td>
        <td
          class=""
        >
          5
        </td>
        <td
          class=""
        >
          6
        </td>
        <td
          class=""
        >
          7
        </td>
        <td
          class=""
        >
          8
        </td>
        <td
          class=""
        >
          9
        </td>
        <td
          class="weekend"
        >
          10
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          11
        </td>
        <td
          class=""
        >
          12
        </td>
        <td
          class=""
        >
          13
        </td>
        <td
          class=""
        >
          14
        </td>
        <td
          class=""
        >
          15
        </td>
        <td
          class=""
        >
          16
        </td>
        <td
          class="weekend"
        >
          17
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          18
        </td>
        <td
          class=""
        >
          19
        </td>
        <td
          class=""
        >
          20
        </td>
        <td
          class=""
        >
          21
        </td>
        <td
          class=""
        >
          22
        </td>
        <td
          class=""
        >
          23
        </td>
        <td
          class="weekend"
        >
          24
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          25
        </td>
        <td
          class=""
        >
          26
        </td>
        <td
          class=""
        >
          27
        </td>
        <td
          class=""
        >
          28
        </td>
        <td
          class=""
        >
          29
        </td>
        <td
          class=""
        >
          30
        </td>
        <td
          class="weekend"
        >
          31
        </td>
      </tr>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`DaysGrid rendered with 2001-4 match snapshot 1`] = `
<DocumentFragment>
  <table>
    <caption>
      April
    </caption>
    <thead>
      <tr>
        <td
          class="weekend"
        >
          Sun
        </td>
        <td
          class=""
        >
          Mon
        </td>
        <td
          class=""
        >
          Tue
        </td>
        <td
          class=""
        >
          Wed
        </td>
        <td
          class=""
        >
          Thu
        </td>
        <td
          class=""
        >
          Fri
        </td>
        <td
          class="weekend"
        >
          Sat
        </td>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td />
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class=""
        >
          3
        </td>
        <td
          class=""
        >
          4
        </td>
        <td
          class="weekend"
        >
          5
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          6
        </td>
        <td
          class=""
        >
          7
        </td>
        <td
          class=""
        >
          8
        </td>
        <td
          class=""
        >
          9
        </td>
        <td
          class=""
        >
          10
        </td>
        <td
          class=""
        >
          11
        </td>
        <td
          class="weekend"
        >
          12
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          13
        </td>
        <td
          class=""
        >
          14
        </td>
        <td
          class=""
        >
          15
        </td>
        <td
          class=""
        >
          16
        </td>
        <td
          class=""
        >
          17
        </td>
        <td
          class=""
        >
          18
        </td>
        <td
          class="weekend"
        >
          19
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          20
        </td>
        <td
          class=""
        >
          21
        </td>
        <td
          class=""
        >
          22
        </td>
        <td
          class=""
        >
          23
        </td>
        <td
          class=""
        >
          24
        </td>
        <td
          class=""
        >
          25
        </td>
        <td
          class="weekend"
        >
          26
        </td>
      </tr>
      <tr>
        <td
          class="weekend"
        >
          27
        </td>
        <td
          class=""
        >
          28
        </td>
        <td
          class=""
        >
          29
        </td>
        <td
          class=""
        >
          30
        </td>
        <td
          class=""
        >
          31
        </td>
        <td />
        <td />
      </tr>
      <tr>
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
        <td />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
